testdata/p4_16_samples/issue383-bmv2.p4

```diff
@@ -50,27 +50,27 @@ struct parsed_packet_t {
 };
 
 struct tst_t {
-  row_t row0;
-  row_t row1;
-  col_t col;  
-  bitvec_hdr bvh0;
-  bitvec_hdr bvh1;
+    row_t row0;
+    row_t row1;
+    col_t col;  
+    bitvec_hdr bvh0;
+    bitvec_hdr bvh1;
 }
 
 parser parse(packet_in pk, out parsed_packet_t h,
-             inout local_metadata_t local_metadata,
-             inout standard_metadata_t standard_metadata) {
-  state start {
-    pk.extract(h.bvh0);
-    pk.extract(h.bvh1);
-    pk.extract(local_metadata.col.bvh);
-    transition accept;
-  }
+inout local_metadata_t local_metadata,
```

Comment:
this indentation is worse than it used to be for these two lines.

Reply:
This file's indentation also uses 4 spaces from the top statement 'state'.